package.json

```diff
@@ -564,6 +565,7 @@
         "@types/ws": "7.2.6",
         "@typescript-eslint/eslint-plugin": "4.19.0",
         "@typescript-eslint/parser": "4.19.0",
+        "copy-webpack-plugin": "^8.1.1",

```

Comment:
let's keep a fixed version. (no ^ [dont know what is the name for this symbol :D] )

Reply:
carot :)

Reply:
Done